README.md

```diff
@@ -45,7 +45,76 @@ Host your own `amazon-connect-chat-interface.js` bundle file and provide the lin
 
 ![Host Widget Snippet Integrated Custom UI](/.github/screenshots/custom-chat-widget-interface-screenshot.png)
 
-### Option 3: Customized Widget and Chat Interface UI (Self-Hosted)
+### Option 3: Customize Widget Start Chat Response
```

Comment:
I believe the name of the option has been updated to `Hosted widget UI with custom Start Chat API`
https://quip-amazon.com/ydPnASb82LDg/Draft-Git-UI-Examples-CustomStartChat-section#temp:C:eHX29a8be29a4ac48a2a4c3a1262